code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -212,7 +212,7 @@ Due to keyboard shortcuts, the second one is not necessarily the remote eye's lo
 		return
 
 	target_mob.visible_message(span_notice("[target_mob] vanishes as [p_theyre()] reclaimed for recycling!"))
-	connected_recycler.use_power(500)
+	connected_recycler.use_energy(500)
```

Comment:
```suggestion
	connected_recycler.use_energy(0.5 KILO JOULES)
```

Reply:
I'll do 500 JOULES when I get on my PC. Same for all other occurences of this.